### PR TITLE
Standing on structures such as crates, tables and bed will now look like it.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
From the creator of 'cosmetic' elements such as footstep_override and immerse...

I've made an element called elevation that nudges mobs a few pixels up while standing on things like tables, crates and beds.

Screenshots of many clones of the same character standing on different objects:
the clone army

## Why It's Good For The Game
It makes it look like the mob is actually standing on an elevated surface.

## Changelog

:cl:
add: Standing on structures such as crates, tables and bed will now look like it.
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]
looks janky on stuff like garbage dumps or beds, like youre standing on the edge of them and about to fall. id move it like 2-3 pixels up

**Review by Wallemations — APPROVED** [review]
idk why this was labelled as a sprite change of all things but overall it's good, maybe some fidgeting with some of the items but otherwise fine.

**Comment by Ghommie** [comment]
> idk why this was labelled as a sprite change of all things but overall it's good, maybe some fidgeting with some of the items but otherwise fine.

Because i've happened to use the image tag in the CL. I'll change that.

File: code/datums/elements/elevation.dm [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	/**
+	 * Since the element can be removed outside of Destroy(),
+	 * and even then, signals are passed down to the new turf,
+	 * it's necessary to clear them here.
+	 */
+	UnregisterSignal(source, list(
+		COMSIG_ATOM_ENTERED,
+		COMSIG_ATOM_AFTER_SUCCESSFUL_INITIALIZED_ON,
+		COMSIG_ATOM_EXITED,
+		COMSIG_TURF_RESET_ELEVATION,
```
Comment (san7890) [inline code/datums/elements/elevation.dm:126]: alphabetize these

**Review by san7890 — CHANGES_REQUESTED** [review]
nit

File: code/datums/elements/elevation.dm (hunk 1 above)
Comment (Mothblocks) [reply code/datums/elements/elevation.dm:126]: @Ghommie Bruh

Comment (Ghommie) [reply code/datums/elements/elevation.dm:126]: Sorry, it's been a bit of a busy week for me.

**Pull request merged**